src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -42,7 +42,7 @@ public static class UriHelper
         /// <param name="query"></param>
         /// <param name="fragment"></param>
         /// <returns></returns>
-        public static string Encode(
+        public static string BuildAbsolute(
             string scheme,
             HostString host,
```

Comment:
> Almost every call into this...
Insufficient data. I see 5 references and the ones that call `new HostString` are samples and tests. The actual code references pull the value from `request.Host` or `HostString.FromUriComponent(uri)`.